service/src/test/java/org/sunbird/learner/actors/UserFrameworkUpdateTest.java

```diff
@@ -0,0 +1,212 @@
+package org.sunbird.learner.actors;
+
+import static akka.testkit.JavaTestKit.duration;
+import static org.junit.Assert.assertTrue;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import akka.actor.ActorRef;
+import akka.testkit.javadsl.TestKit;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.content.util.ContentStoreUtil;
+import org.sunbird.learner.util.DataCacheHandler;
+import org.sunbird.learner.util.Util;
+import org.sunbird.user.dao.impl.UserExternalIdentityDaoImpl;
+import org.sunbird.user.service.impl.UserServiceImpl;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({DataCacheHandler.class, ContentStoreUtil.class, UserServiceImpl.class, Util.class})
+public class UserFrameworkUpdateTest extends UserManagementActorTest {
+
+  private static final String userId = "testUserId";
+  private static UserExternalIdentityDaoImpl userExtDao;
```

Comment:
Why mock our internal class -> u can mock cassandra operation, encryptionService for this purpose, no need to mock internal classes